src/RazorBlade.Library/HtmlHelper.cs

```diff
@@ -51,6 +51,7 @@ public string Encode(object? value)
                 '<'   => "&lt;",
                 '>'   => "&gt;",
                 '"'   => "&quot;",
+                "\'"  => "&#39;",
```

Comment:
- That would need to be `'\''` (match on a char instead of a string)
- Why `&#39;` instead of `&apos;`?